rust/kernel/driver.rs

```diff
@@ -440,3 +440,76 @@ macro_rules! module_driver {
         }
     }
 }
+
+/// Implements a get_id_info function for drivers.
+///
+/// Some linux drivers, e.g. Platform drivers, need to get info from of_id. This macro helps implements the get_id_info
+/// function for those drivers.
```

Comment:
```suggestion
/// Some kinds of drivers, e.g. platform drivers, need to get info from `of_id`.
/// This macro implements a function to do so with signature:
///
/// ```
/// fn get_id_info(dev: &Device) -> Option<&'static T::IdInfo>;
/// ```
```